Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -39,37 +39,41 @@ namespace PFunctor
 instance : Inhabited PFunctor :=
   ⟨⟨default, default⟩⟩
 
-variable (P : PFunctor) {α β : Type u}
+variable (P : PFunctor.{u}) {α : Type v₁} {β : Type v₂}
 
 /-- Applying `P` to an object of `Type` -/
-def Obj (α : Type*) :=
-  Σx : P.A, P.B x → α
+@[coe]
+def Obj (α : Type v) :=
```

Comment:
I'm curious, what is your motivation for changing `Type*` to explicitly giving the universe variable?

Reply:
I want to fix the order of universe variables as `u, v`.
With `Type*`, the order of universe variables becomes `v, u`.